src/shared/icons/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+type IconProps = {
+  width: string;
+  height: string;
+};
+
+const SVGSearch: React.FC<IconProps> = (props: IconProps) => {
```

Comment:
please rename it to `SearchIcon` to be more clear. and the same for another one icon.